emoji-substitution/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "manifest_version": 2,
+  "applications": {
+    "gecko": {
+      "strict_min_version": "49.0.1"
```

Comment:
It could go way below that, Firefox supports let and const the way the spec suggests since Firefox 36 and has had them since a way longer time (see also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#Browser_compatibility). I'd assume the hard limit is more about when WebExtensions were introduced...